src/App.vue

```diff
@@ -34,13 +38,21 @@ import SbcLoader from 'sbc-common-components/src/components/SbcLoader.vue'
   components: {
     SbcHeader,
     SbcFooter,
-    SbcLoader
+    SbcLoader,
+    LoaderComponent
   }
 })
 export default class App extends Vue {
   private showLoading = true
   private logoutUrl = ''
 
+  /* Getter will return true incase of Axios operations (eg POST)
+  This value is used to toggle between showing route and loading progress components */
+  get isLoadingState (): boolean {
+    const store: any = this.$store.state
```

Comment:
can we use vuex-helper in this place instead of this.store. may be we can refactor later